clients/python/girder_client/cli.py

```diff
@@ -97,10 +97,14 @@ def sendRestRequest(self, *args, **kwargs):
         with self.session() as session:
             session.verify = self.sslVerify
             if self.retries:
-                session.mount(self.urlBase, HTTPAdapter(
-                    max_retries=Retry(total=self.retries, backoff_factor=1,
-                                      method_whitelist=False)
-                ))
+                # create a retry handler that:
+                #  1. uses exponential backoff
+                #  2. retries on all methods
+                #  3. retries on default status codes (503, 413, and 429)
```

Comment:
I realize this is an upstream decision, but I'm curious about 413... is that something services typically send back due to high load?

Reply:
The spec says this

> The HTTP 413 Payload Too Large response status code indicates that the request entity is larger than limits defined by server; the server might close the connection or return a Retry-After header field.

I don't know if urllib3 looks at the `Retry-After` header, but since girder doesn't use 413 I don't think it matters much.

Reply:
From the docs about 413:

> If the condition is temporary, the server SHOULD generate a Retry-After header field to indicate that it is temporary and after what time the client MAY try again.

`Retry` has a parameter:

```
:param bool respect_retry_after_header:
 |      Whether to respect Retry-After header on status codes defined as
 |      :attr:`Retry.RETRY_AFTER_STATUS_CODES` or not.
```

Which defaults to `True`.

As such, is it prudent to specify `status_forcelist`?